internal/app/errors.go

```diff
@@ -27,4 +27,5 @@ var (
 	ErrHistoryIDRequired       = errors.New("history id is required")
 	ErrAliasRequired           = errors.New("alias is required")
 	ErrAliasAndIDNotAllowed    = errors.New("alias and id bith specified, only 1 is allowed")
+	ErrNoHistoryFound          = errors.New("no history entry found")
```

Comment:
We could probably use the error already defined [here](https://github.com/fidelity/kconnect/blob/main/pkg/history/store.go#L32)?